tests/e2e/functional_test.go

```diff
@@ -218,33 +215,29 @@ var _ = Describe("PipelineRollout e2e", func() {
 
 		// new NumaflowController spec
 		updatedNumaflowControllerSpec := apiv1.NumaflowControllerRolloutSpec{
-			Controller: apiv1.Controller{Version: "1.1.7"},
+			Controller: apiv1.Controller{Version: "0.0.6"},
```

Comment:
this is one of the latest RC manifests for numaflow, to bring in all of the changes